src/reference-guide/controller-api/v2.0.md

```diff
@@ -337,7 +337,8 @@ DEIS_API_VERSION: 2.0
 DEIS_PLATFORM_VERSION: 1.12.1
```

Comment:
This version is actually `2.0.0-dev`, but that's out of scope for this PR. I'll make an issue for that.